home-page/.expo/packager-info.json

```diff
@@ -0,0 +1,6 @@
+{
+  "expoServerPort": null,
+  "expoServerNgrokUrl": null,
+  "packagerNgrokUrl": null,
+  "ngrokPid": null
+}
```

Comment:
This red icon means that there was no extra newline at the end of your file. Just add a newline and you should be all set.